test/build.info

```diff
@@ -685,7 +685,7 @@ IF[{- !$disabled{tests} -}]
     DEPEND[keymgmt_internal_test]=../libcrypto.a libtestutil.a
 
     SOURCE[ffc_internal_test]=ffc_internal_test.c
-    INCLUDE[ffc_internal_test]=.. ../include ../apps/include ../crypto/include
+    INCLUDE[ffc_internal_test]=.. ../include ../apps/include ../include/crypto
```

Comment:
Without having tested it, I'm quite sure that you can omit the `../include/crypto` directory entirely, for two reasons:

Short reason:  It compiled without, so why add it?

Detailed reason: one of the goals of the refactoring was to reduce the amount of private include directory paths to only one, namely `../include` and have relative includes like the following instead. 

```C
#include "crypto/evp.h"
```
That's why `../include/crypto` is not necessary.

This approach had also the positive side effect to disambiguate between _private_ include directives (including header files from ../include, shared by several libcrypto subsystems) and _local_ include directives (including header files located in the same subdirectory as the C file).

So in fact, keeping the path `../include/crypto` is harmful, because it restores ambiguity: the same file could now be included as 

```C
#include "evp.h"
```

Reply:
+1

Reply:
Fixed